fabric-structure-api-v1/src/main/java/net/fabricmc/fabric/api/structure/v1/FabricStructurePool.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.structure.v1;
+
+import com.mojang.datafixers.util.Pair;
+
+import net.minecraft.structure.pool.StructurePool;
+import net.minecraft.structure.pool.StructurePoolElement;
+
+import net.fabricmc.fabric.mixin.structure.StructurePoolAccessor;
+
+/**
+ * Represents a modifiable structure pool that would have several helper methods for modders.
+ */
+public class FabricStructurePool {
+	private final StructurePool underlying;
+
+	public FabricStructurePool(StructurePool underlying) {
+		this.underlying = underlying;
+	}
+
+	/**
+	 * Adds a new {@code StructurePoolElement} to the {@code StructurePool}.
+	 * See the alternative {@linkplain #addStructurePoolElement(StructurePoolElement, int)}  for details.
```

Comment:
```suggestion
	 * See the alternative {@link #addStructurePoolElement(StructurePoolElement, int)} for details.
```
Better as a non-plaintext link